### PR TITLE
INBUS integration to get user info

### PR DESCRIPTION
**Pull request by geordi**
Partially solves #193 .

This if the first INBUS API call we can make. More to come.

### CODE REVIEW

File: common/inbus.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    return token
+
+
+def request_new_token() -> Dict:
+    token = authenticate()
+    return token
+
+
+def is_response_ok_or_new_token_(response: requests.Response) -> bool:
+    if response.status_code == 404:
```
Comment (Kobzol) [inline common/inbus.py:54]: It returns 404 when authentication is not correct? Nice.. :)

File: common/inbus.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    if not person_resp:
+        return None
+    person_json = person_resp.json()
+
+    return person_json
+
+
+# Kelvin's interface
+
+def search_user(login: str) -> Dict:
```
Comment (Kobzol) [inline common/inbus.py:98]: Could this return a dataclass instead? We pass this data to the constructor of `User` through `**`, that is quite brittle :)

**Review by Kobzol — APPROVED** [review]

File: common/inbus.py (hunk 1 above)
Comment (geordi) [reply common/inbus.py:54]: Sorry, my bad. It's `401` actually. I'll fix it in next commit.

**Review by Kobzol — APPROVED** [review]
LGTM

**Pull request merged**